app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
@bzwei @kevensen I don't understand what's the benefit of opt 2 here. In fact:

1. it's not catching all the possible exceptions (and I am not suggesting to do it)
2. it's masking the original exception making debugging harder
3. recovery is impossible anyway

I think that opt 2 should be left only for those cases where it actually serve something.
@bzwei what would happen if the `KubeException` exception reaches the automation engine? How `MiqProvisionError` would make it better?